bin/ci-push.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Push output back to GitHub
+
+
+cd output || exit 1
+
+git config --global user.email "${COMMIT_EMAIL:-travis@travis-ci.org}" || exit 1
```

Comment:
you should probably also change the email and commit name info?

Reply:
I wasn't sure how we'll want to go about doing the switch, so this script was left in a state where it's possible to run from both Travis and GitHub Actions.

In GitHub Actions, they'll be overridden by these `COMMIT_*` environment variables: https://github.com/Jermolene/TiddlyWiki5/pull/4793/files#diff-e9f950f17198d3d5e3122a44230a09b9R8-R10

[Travis doesn't set these `COMMIT_*` environment variables](https://docs.travis-ci.com/user/environment-variables#default-environment-variables) (just checked now, phew), so the string after `:-` will be used instead.